CONTRIBUTING.md

```diff
@@ -66,97 +67,32 @@ clients that need to extract a single description/example.
 
 ## Submitting a pull request
 
-TL;DR: fork, clone, `npm install`, feature branch, commit, push, pull request, check Travis.
+For submitting changes, you can use whatever workflow you're more comfortable with.
+The easiest way is to just edit the page directly on the Github interface.
+Check out the step-by-step instructions (with screenshots) on
+[Github Help](https://help.github.com/articles/editing-files-in-another-user-s-repository/).
 
-Detailed explanation:
+Alternatively, you can do it using the command line:
```

Comment:
>   for a newcomer the web-based workflow would be easier

agree.

> I'm thinking of doing something like this: https://www.mediawiki.org/wiki/Gerrit/Getting_started

This is also okay. I like the section wise approach the article takes.
- Get the code
```
some git commands
```
- Modify the code
```
some git commands
```
- Send a pull request
```
more commands
```
Of course, a lot needs to be simplified. The git commands shown in gerrit are too deep for a newbie.